api-tests/features/repeat-fraud-check/p2-repeat-fraud-check.feature

```diff
@@ -1,138 +1,214 @@
+      When I submit a 'update-name' event
+      Then I get a 'dcmaw' CRI response
+      When I submit 'kenneth-changed-given-name-driving-permit-valid' details to the CRI stub
+      Then I get a 'page-dcmaw-success' page response with context 'coiAddress'
+
+    Scenario: Unsupported Changes
+      # Repeat fraud check with various unsupported events and back navigation
+      When I submit a 'dob' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'address-dob' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'dob-family' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'dob-given' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'family-given' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'address-family-given' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'address-dob-family-given' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'address-dob-family' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'address-dob-given' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+      When I submit a 'back' event
+      Then I get a 'confirm-your-details' page response
+      When I submit a 'address-family-given' event
+      Then I get a 'update-name-date-birth' page response with context 'rfcAccountDeletion'
+
+  Rule: Match M1C
+
+    Background:
+      Given the subject already has the following credentials
+        | CRI     | scenario               |
+        | dcmaw   | kenneth-passport-valid |
+        | address | kenneth-current        |
+      And the subject already has the following expired credentials
+        | CRI   | scenario        |
+        | fraud | kenneth-score-2 |
```

Comment:
It doesn't make any difference to whether the test passes, but to be realistic I think this should also be `kenneth-no-applicable`
People should only be getting this response for international addresses and we aren't changing the address in all the test that use this.

Reply:
Could also have tests for users changing to an international address I guess, but that's probably overkill

Reply:
Oh I see your point, in some cases they could have moved abroad but really the tests don't always have changed address.